Description: Fix class names in host monitor
 Use fully qualified class names, with package name prepended.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/hostmon/HostInfoRetriever.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoRetriever.java	2010-03-11 20:55:56.490753683 +0100
+++ airport-utils-2/hostmon/HostInfoRetriever.java	2010-03-11 20:58:12.644752985 +0100
@@ -40,7 +40,9 @@
     static 
     {
         Class retrieverClass;
-        String[] retrieverClassNames = {"AirportExtremeHostInfoRetriever", "SnowAirportHostInfoRetriever", "GraphiteAirportHostInfoRetriever"};
+        String[] retrieverClassNames = {"airporthostmon.AirportExtremeHostInfoRetriever",
+					"airporthostmon.SnowAirportHostInfoRetriever",
+					"airporthostmon.GraphiteAirportHostInfoRetriever"};
         
         for (int i = 0; i < retrieverClassNames.length; i++)
         {
